docs/glossary.md

```diff
@@ -0,0 +1,84 @@
+# OLM Glossary
+
+A list of OLM terms, with definitions and common Aliases.
+
+## CustomResourceDefinitions (CRDs)
+
+The following [CustomResourceDefinitions](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) are defined by the OLM.
+
+### ClusterServiceVersion
+
+**Definition**: The ClusterServiceVersion is a required resource for an Operator which includes application metadata such as name, description, version, repository link, labels, icon, etc. It also includes a list of corresponding/required resources and their statuses, install strategy of the Operator as [deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/), supported install modes, etc.
+**Aliases**: CSV(s)
+
+### CatalogSources
+
+**Definition**: The CatalogSource represents a repository of bundles, which are collections of resources that must contain [CSVs](#ClusterServiceVersion), [CRDs](#CustomResourceDefinitions), and package definitions. There are multiple implementations of a CatalogSource backend, the current recommendation is to use a [registry image](#Index).
+**Aliases**: CatSrc(s)
+
+### InstallPlans
+
+**Definition**: The InstallPlan defines a set of resources to be created in order to install or upgrade to a specific version of [CSV](#ClusterServiceVersion). 
+**Aliases**: IP(s)
+
+### OperatorGroups
+
+**Definition**: The OperatorGroup selects a set of target namespaces in which to generate required RBAC access for its member Operators.
+**Aliases**: OG(s)
+
+### OperatorSources
+
+**Definition**: The OperatorSources are external app registries that contains a catalog of operators. Applying an OperatorSource to a cluster makes the operators in that OperatorSource available for installation in that cluster.
+**Aliases**: OpSrc(s)
+
+### Subscriptions
+
+**Definition**: The Subscription defines the channel and the source of Operator updates and is used to keep the CSV updated by binding with a channel in a package. 
+**Aliases**: Subs(s)
+
+## OLM Concepts
+
+### Bundle
+
+**Definition**: A collection of Operator [CSV](#ClusterServiceVersion), manifests, and metadata which together form a unique version of an Operator that can be installed onto the cluster. 
+**Aliases**:
+
+### Bundle Image
+
+**Definition**: An image of a bundle is built from operator manifests and contains exactly one [bundle](#Bundle). The bundle images are stored and distributed by app registries such as Quay.io or DockerHub.
```

Comment:
`app registries` should be `OCI spec container registries`

Reply:
Do we want to say OCI though? how about just `container registries`. Kinda do not want to also spell out Open Container Initiative. I'll leave OCI up there as you suggested though, coz it's cool.